README.md

```diff
@@ -18,10 +18,10 @@ installed and their bin folders have been added to `PATH`.
 
 To ensure gcloud was authorized to access the cloud project and we have the
 `app` component installed, we assume the following has been run:
-```
-% gcloud auth login
-% gcloud config set project <project-name>
-% gcloud components update app
+```console
+$ gcloud auth login
```

Comment:
This time I explicitly removed the dollar signs, because they have a different meaning in a shell (namely env variables).

My shell prompt btw looks like:

   kustermann@kustermann-macbookpro2 ~ %